### ISSUE TITLE
ID 258651: [Keyboard Navigation - Simon – Value Mapping]: Focus order does not return to “Set color” button after pressing esc key.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#72807]
### **User Experience**:  
Users who rely on the keyboard will be affected if the keyboard focus is lost after invoking the ‘ESC' key on. Due to this issue, the user will be misguided and unable to track the keyboard focus.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**  
1. Open #Url: [play.grafana.org](https://play.grafana.org/)  Sign in with valid credentials. "Home" page appears. 
2. Navigate the Feature Showcase pane heading and select it. 
3. Navigate the expanded Dropdown and select the "Edit" menu item.  
4. Navigate the page "Edit" page which appears.  
5. Navigate to the "All" buttons and select it. 
6. Navigate to "Value Mapping" button and expand it.  
7. Navigate all the controls present under it. 
8. Verify the issue

**Actual Result:** 
Issue: Focus order does not return to “Set color” button pressing esc key.
Observation: When focus is inside the set color Dialog,after invoking 'ESC' key focus order disappears.

**Expected Result:**

**Solution:** 
Focus order should return to Setcolor button after pressing esc key. EX: When focus is inside the Set color dialog, after invoking 'ESC'key focus order should retain back to the Set color button.
_Note:  
Same issue appears with same control 'Color dialog' throughout the application. 
Same issue repro 'Click to view alternative operations' button present in "Explore" page. 
Same issue exists with 'Save' button after pressing close button focus does not return back to save button. (Edit>Sav)._

Severity:
`SEV3`

Area:
`Not known`

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:
